### PR TITLE
Fix up docblocks for assoc options and list arrays.

### PR DESCRIPTION
**Pull request by dereuromark**
I thought I already did all of those back in the days
Apparently, not.

### CODE REVIEW

File: src/Controller/Controller.php [hunk 1]
```diff
@@ -728,9 +728,9 @@ public function viewClasses(): array
      * Each view class must implement the `getContentType()` hook method
      * to participate in negotiation.
      *
-     * @param array $viewClasses View classes list.
+     * @param array<string> $viewClasses View classes list.
```
Comment (othercorey) [inline src/Controller/Controller.php]: Think we can use list<string> now.

Comment (dereuromark) [reply src/Controller/Controller.php]: True, we can update accordingly.

Comment (dereuromark) [reply src/Controller/Controller.php]: I found also a few others
There are potentially 87 other occurances left, but thats maybe for a follow up PR.

**Comment by dereuromark** [comment]
We should sqash merge this

**Review by LordSimal — APPROVED** [review]

**Pull request merged**